std/experimental/logger/filelogger.d

```diff
@@ -200,3 +227,20 @@ class FileLogger : Logger
     assert(tl !is null);
     stdThreadLocalLog.logLevel = LogLevel.all;
 }
+
+@safe unittest
+{
+    import std.file : exists, rmdirRecurse;
+
+    string logpath = "super/long/path/";
```

Comment:
Please prepend with `std.file.deleteme`, don't just create test files in the local directory.